ir-black-theme.el

```diff
@@ -5,7 +5,8 @@
 ;; Author: Jon-Michael Deldin <dev@jmdeldin.com>
 ;; Keywords: faces
 ;; Compatibility: 24
-;; Version: 1.0.1
+;; Version: 20130302.2355
+;; X-Original-Version: 1.0.1
```

Comment:
I think you can omit the version bump -- I will do that in a separate commit to master.